experimental/Schemes/BlowupMorphism.jl

```diff
@@ -98,3 +111,76 @@ end
 @attr function dim(I::MPolyLocalizedIdeal)
   return dim(saturated_ideal(I))
 end
+
+@Markdown.doc """
+    strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
+
+For a `BlowupMorphism` ``p : Y → X`` and a `CoveredClosedEmbedding` 
+``ι : Z ↪ X``, compute the strict transform ``Z'`` of ``Z`` along ``p`` and 
+return the induced projection ``p : Z' → Z`` as 
+an ``AbsCoveredSchemeMorphism``.
+"""
+function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
+  Y = domain(p)
+  X = codomain(p)
+  Z = domain(inc)
+  codomain(inc) === X || error("maps must have the same codomain")
+  ID = IdDict{AbsSpec, Ideal}()
+  pr = projection(p)
+  p_cov = covering_morphism(pr)
+  CY = domain(p_cov)
+  # We first apply elim_part to all the charts.
+  #CY_simp = simplified_covering(Y)
+  #phi = Y[CY_simp, CY]
+  CY_simp, phi, psi = simplify(CY)
+  # register the simplification in Y
+  push!(coverings(Y), CY_simp)
+  refinements(Y)[(CY_simp, CY)] = phi
+  refinements(Y)[(CY, CY_simp)] = psi
+  p_cov_simp = compose(phi, p_cov)
+  CX = codomain(p_cov)
+  E = exceptional_divisor(p)
+  for U in patches(CY_simp)
+    p_res = p_cov_simp[U]
+    V = codomain(p_res)
+    J = image_ideal(inc)(V)
+    pbJ = ideal(OO(U), pullback(p_res).(gens(J)))
+    pbJ_sat = saturated_ideal(pbJ)
+    pbJ_sat = saturation(pbJ_sat, ideal(base_ring(pbJ_sat), lifted_numerator.(E(U))))
+    pbJ = ideal(OO(U), [g for g in OO(U).(gens(pbJ_sat)) if !iszero(g)])
+    ID[U] = pbJ
+  end
+
+  I_trans = IdealSheaf(Y, ID, check=true) # TODO: Set to false
+  inc_Z_trans = CoveredClosedEmbedding(Y, I_trans, covering=CY_simp)
+  Z_trans = domain(inc_Z_trans)
```

Comment:
Why do you create Z_trans, if you do not use it afterwards?

Reply:
Maybe this line is superfluous. I don't remember now. But in any case: It's not created in line 156. The creation takes place already in line 155. And I'm pretty sure, I am using that map in what follows. So no severe harm should have been done.